nsv13/code/modules/munitions/ammunition/torpedos/torpedo_parts.dm

```diff
@@ -1,14 +1,19 @@
 /obj/item/ship_weapon/parts/missile/warhead/torpedo
-	name = "NTP-2 guided torpedo payload"
-	desc = "A payload that makes a standard torpedo payload"
+	name = "NTP-2 standard torpedo payload"
 	icon_state = "warhead"
-	desc = "An extremely heavy warhead designed to be fitted to a torpedo. This one has an inbuilt plasma charge to amplify its damage."
+	desc = "An extremely heavy warhead designed to be fitted to a torpedo. This one contains a standard explosive charge."
 	fits_type = /obj/item/ship_weapon/ammunition/torpedo/torpedo_casing
 	build_path = /obj/item/ship_weapon/ammunition/torpedo
 
+/obj/item/ship_weapon/parts/missile/warhead/decoy
+	name = "NTP-0x 'DCY' electronic countermeasure torpedo payload"
+	icon_state = "warhead_decoy"
+	desc = "A simple electronic countermeasure wrapped in a metal casing. While these form inert missiles, they can be used to distract enemy anti-missile defenses to divert their flak away from other targets."
+	fits_type = /obj/item/ship_weapon/ammunition/torpedo/torpedo_casing
```

Comment:
I personally think the decoys should fit missile frames (like the description says) but right now they fit torps and their name says torp so either edit the description or name/functionality.

Reply:
Missed that missile. Anyway, making it fit both would be out of scope for this PR so I just changed the desc.